### ISSUE TITLE
Installation error - 1193: Unknown system variable 'storage_engine'

### ISSUE THREAD

**Issue by purplechap** [symphonycms/symphony-2#2587]
I am running the Symphony install and receive the error below. 

Here is my configuration.
Ubuntu 16.04
Apache 2.4.18
PHP 7.0.4-7
MySQL 5.7.11
Symphony 2.6.7 installed from GitHub

```
============================================
INSTALLATION PROCESS STARTED (2016-04-22T23:43:59-06:00)
============================================
04/22/2016 11:43 pm > Notice: MYSQL: Establishing Connection
04/22/2016 11:43 pm > Notice: MYSQL: Importing Table Schema
04/22/2016 11:43 pm > Fatal Error: There was an error while trying to import data to the database. MySQL returned: 1193: Unknown system variable 'storage_engine'
============================================
INSTALLATION ABORTED: Execution Time - 1 sec (22.04.16 23:43:59)
============================================
```

After some digging around it appears that storage_engine was deprecated in MySQL 5.7.5 [storage_engine](http://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_storage_engine)

If this version of MySQL is unsupported please let me know and I'll downgrade my MySQL, but I did follow the requirement "MySQL 5.5 or above".

Regards, Mike.

**Comment by nitriques** [comment]
PHP7 is not working at the moment. Can you please try with [this tree](https://github.com/symphonycms/symphony-2/tree/0abd8401fd7469d60dc1c1e626622db3118a3ce4) ?

**Comment by brendo** [comment]
While PHP 7 support needs to be ironed out (and we can expect a lot more of these reports with Ubuntu 16's release), the fix for MySQL 5.7 has been committed to the 2.7.x branch in [this commit](https://github.com/symphonycms/symphony-2/commit/9363d504eea87579f9825fd42694887b781a1795)

**Comment by findborg** [comment]
I am having the same problem

**Comment by nitriques** [comment]
@findborg even with the `2.7.x` branch ?

**Comment by findborg** [comment]
Yes, here is my local setup.
Windows 10 using wamp64
Server: Local Databases (#### via TCP/IP)
Server type: MySQL
Server version: 5.7.9 - MySQL Community Server (GPL)
Protocol version: 10
User: root@localhost
Server charset: UTF-8 Unicode (utf8)
Apache/2.4.17 (Win64) PHP/5.6.16
Database client version: libmysql - mysqlnd 5.0.11-dev - 20120503 - $Id: #### $
PHP extension: mysqli Documentation
PHP version: 5.6.16

**Comment by findborg** [comment]
Here is the log:
Log Created: 2016-06-21T08:22:25+00:00
============================================
Symphony Log
Opened:  2016-06-21T08:22:25+00:00
Version: 2.6.5
Domain:  localhost/blogger
--------------------------------------------

============================================
INSTALLATION PROCESS STARTED (2016-06-21T08:23:38+00:00)
============================================
06/21/2016 8:23 am > Notice: MYSQL: Establishing Connection
06/21/2016 8:23 am > Notice: MYSQL: Importing Table Schema
06/21/2016 8:23 am > Fatal Error: There was an error while trying to import data to the database. MySQL returned: 1193: Unknown system variable 'storage_engine'
============================================
INSTALLATION ABORTED: Execution Time - 1 sec (21.06.16 08:23:38)
============================================

**Comment by brendo** [comment]
@findborg The fix should be this commit, could you apply it and try again? https://github.com/symphonycms/symphony-2/commit/9363d504eea87579f9825fd42694887b781a1795

Based off the log, you are using 2.6.5, not 2.7.x

**Comment by findborg** [comment]
Just noticed that myself,  give me a sec ant I will try your link

**Comment by findborg** [comment]
Ok now this is the result:

============================================
INSTALLATION PROCESS STARTED (2016-06-21T08:32:23+00:00)
============================================
06/21/2016 8:32 am > Notice: MYSQL: Establishing Connection
06/21/2016 8:32 am > Notice: MYSQL: Importing Table Schema
06/21/2016 8:32 am > Fatal Error: There was an error while trying to import data to the database. MySQL returned: 1067: Invalid default value for 'last_seen'
============================================
INSTALLATION ABORTED: Execution Time - 1 sec (21.06.16 08:32:23)
============================================

**Comment by brendo** [comment]
Heh, nice, this is now #2594, and I believe this comment will be useful to you, https://github.com/symphonycms/symphony-2/issues/2594#issuecomment-221155459

**Comment by findborg** [comment]
I have gotten this far before,  I can do a manual sql query on the sql file and all the tables are created,  but when I try with the installer it errors out straight off
I will try 2594

**Comment by brendo** [comment]
This is the actual commit that fixes it, https://github.com/symphonycms/symphony-2/commit/db403c3d9865d363b79199c5671ac75bf3396d2d

Sorry for the commits, we're working on MySQL 5.7 strict mode support in the 2.7.x branch which is not released yet!

**Comment by findborg** [comment]
The floor is yours

**Comment by nitriques** [comment]
:+1:

**Comment by nitriques** [comment]
Sorry to not have edited the commit message and linked to here :-1:

**Comment by timokleemann** [comment]
Just as a reference to myself: Replacing `storage_engine` with `default_storage_engine` in `symphony/lib/toolkit/class.mysql.php` usually fixes the problem and installs the latest version of Symphony on MySQL 5.7.

**Comment by nitriques** [comment]
Fixed in 2.7.0, closing.
